packages/editor/src/components/preview-dropdown/index.js

```diff
@@ -110,13 +109,6 @@ export default function PreviewDropdown( { forceIsAutosaveable, disabled } ) {
 	 */
 	const onSelect = ( value ) => {
 		setDeviceType( value );
-		if ( value === 'Desktop' ) {
-			speak( __( 'Desktop selected' ), 'assertive' );
-		} else if ( value === 'Tablet' ) {
-			speak( __( 'Tablet selected' ), 'assertive' );
-		} else {
-			speak( __( 'Mobile selected' ), 'assertive' );
-		}
 	};
```

Comment:
Could you remove this `onSelect` and inline it in the `<MenuItemsChoice>` `onSelect`? Like:

```
onSelect={ ( value ) => {
	setDeviceType( value );
} }
```

You might even be able to do:

```
onSelect={ setDeviceType }
```

Reply:
@jeryj updated as you suggested.